devfiles/no-custom-image/devfile.yaml

```diff
@@ -0,0 +1,81 @@
+schemaVersion: 2.0.0
+metadata:
+  version: 0.1.0
+  description: Java application devfile using Open Liberty runtime
+  name: java-openliberty
+starterProjects:
+  - name: user-app
+    git:
+      remotes:
+        origin: 'https://github.com/OpenLiberty/application-stack-starters.git'
+components:
+  - name: devruntime
+    container:
+      image: maven:3.6-adoptopenjdk-11-openj9
+      memoryLimit: 1512Mi
+      mountSources: true
+      endpoints:
+        - exposure: public
+          path: /
+          name: ep1
+          targetPort: 9080
+          protocol: http
+      volumeMounts:
+        - name: m2
+          path: /home/user/.m2
+  - name: m2
+    volume:
+      size: 3Gi
```

Comment:
Just curious how we got to 3G.   What is the size if you look at our intro sample, eg..?

Reply:
This sample was based on the existing m2-repo-volume/devfile.yaml as a requested pre-req. I am not aware of the details as to why that value was chosen. Perhaps @wisniew90 can answer your question.